src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  width: getContainerSize(
+    $calendar-padding,
+    $days-gap,
+    $days-size,
+    $days-columns
+  );
+  padding: 20px $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+
+  @for $index from 28 through 31 {
+    &--month-length_#{$index} >:nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  @each $index, $day in $days {
+    &--start_#{$day}>:first-child {
+      margin-left: $days-size * $index;
+    }
+  }
+
+  @for $index from 1 through $month-length {
+    &>:nth-child(#{$index})::before {
+      content: "#{$index}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      display: flex;
+      position: relative;
+      top: 50%;
+      transform: translate(0, -50%);
+      justify-content: center;
+    }
+  }
+
+  &__day {
+    transition-duration: 500ms;
+
+    &:nth-child(n) {
+      width: $days-size;
+      height: $days-size;
+      border: 1px solid black;
+      background-color: #eee;
+    }
+  }
+
+  &__day:hover {
+    background-color: pink;
+    transform: translate(0, -20px);
+    justify-content: center;
```

Comment:
you already have this in `&__day`

Reply:
@ivanchuk-yurii  sorry didn't get it, there is no same properties in these two selectors as I see.